app/src/components/Footer/Footer.tsx

```diff
@@ -1,30 +1,58 @@
-import React from 'react'
-import styled from 'styled-components'
+import React, { useEffect } from 'react'
+import styled, { keyframes } from 'styled-components'
+import { useBlockNumber } from '@/hooks/data'
 
 import Nav from './components/Nav'
 
-const Footer: React.FC = () => (
-  <StyledFooter>
-    <StyledFooterInner>
-      <Nav />
-    </StyledFooterInner>
-  </StyledFooter>
-)
+const Footer: React.FC = () => {
+  const { data } = useBlockNumber()
+  return (
+    <StyledFooter>
+      <StyledFooterInner>
+        <Nav />
+      </StyledFooterInner>
+      <StyledBlockNumber>
+        {data}
+        <Loader />
+      </StyledBlockNumber>
+    </StyledFooter>
+  )
+}
 
+const breathe = keyframes`
+  0% { opacity: .5;}
+  50% {opacity: 1; }
+  100% { opacity: .5;}
+;`
+const Loader = styled.div`
+  border-radius: 50%;
+  width: 10px;
+  margin-left: 1em;
+  height: 10px;
+  background-color: ${(props) => props.theme.color.white};
+  color: ${(props) => props.theme.color.white};
+  animation: ${breathe} 2s infinite;
+`
+const StyledBlockNumber = styled.h5`
+  display: flex;
+  align-items: center;
+  color: ${(props) => props.theme.color.white};
+`
 const StyledFooter = styled.footer`
   align-items: center;
-  bottom: 0;
+  bottom: 1em;
   display: flex;
-  justify-content: center;
+  padding-left: 1em;
+  padding-right: 2em;
+  justify-content: space-between;
 `
 const StyledFooterInner = styled.div`
   align-items: center;
   display: flex;
   justify-content: center;
   height: ${(props) => props.theme.topBarSize}px;
   max-width: ${(props) => props.theme.siteWidth}px;
-  width: 100%;
-  position: fixed;
+
   margin-right: ${(props) => props.theme.spacing[4]}px;
   padding: ${(props) => props.theme.spacing[4]}px;
 `
```

Comment:
Note for me:
- alphabetize
- conditional styling

Reply:
Should probably fix the footer to the bottom, shouldn't have to scroll down to see anything.